### PR TITLE
850 feature async rorder pages

### LINKED ISSUE #850: Move Remove pages fragment parse operation to async thread
Move the parse function: 
https://github.com/Swati4star/Images-to-PDF/blob/master/app/src/main/java/swati4star/createpdf/fragment/RemovePagesFragment.java#L189
to background thread

In the meanwhile, show the animation dialog (like in images to PDF)

### PR DESCRIPTION
**Pull request by tanmay23235616**
# Description

Made reordering of pages an asynchronous task. 

Fixes #850 

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/util/PDFUtils.java [hunk 1]
```diff
@@ -269,4 +276,89 @@ public boolean reorderRemovePDF(String inputPath, String output, String pages) {
...
+    /**
+     * @param uri Uri of the pdf
+     * @param path Absolute path of the pdf
+     * @param onPdfReorderedInterface interface to update  pdf reorder progress
+     * */
+    public void reorderPdfPages(Uri uri, String path, @NonNull OnPdfReorderedInterface onPdfReorderedInterface) {
+        new ReorderPdfPagesAsync(uri, path, mContext, onPdfReorderedInterface).execute();
+    }
+
+    @SuppressLint("StaticFieldLeak")
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/PDFUtils.java]: dont supress this lint warning

File: app/src/main/java/swati4star/createpdf/util/DialogUtils.java [hunk 2]
```diff
@@ -13,6 +21,9 @@
     public static final int REMOVE_PASSWORD = 22;
     public static final int ADD_WATERMARK = 23;
 
+    @BindView(R.id.textView)
+    public TextView mDialogTitle;
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/DialogUtils.java]: `dialogTitle`

File: app/src/main/java/swati4star/createpdf/util/DialogUtils.java [hunk 3]
```diff
@@ -1,9 +1,17 @@
 package swati4star.createpdf.util;
 
 import android.app.Activity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.TextView;
 
 import com.afollestad.materialdialogs.MaterialDialog;
 
+import org.w3c.dom.Text;
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/DialogUtils.java]: why is this import needed?

File: app/src/main/java/swati4star/createpdf/util/PDFUtils.java [hunk 4]
```diff
@@ -269,4 +276,89 @@ public boolean reorderRemovePDF(String inputPath, String output, String pages) {
...
+    private static class ReorderPdfPagesAsync extends AsyncTask<String, String, ArrayList<Bitmap>> {
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/PDFUtils.java]: does this class have to be static?

**Comment by tanmay23235616** [comment]
> Please do the requested changes

Done the changes
